src/components/GithubProfile.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Box,
+  Link,
+  Heading,
+  Spinner,
+  Grid,
+  Flex,
+  Icon,
+  useColorModeValue,
+  IconButton,
+  Stat,
+  StatLabel,
+  StatNumber,
+} from '@chakra-ui/react';
+import { FaGithub, FaExternalLinkAlt } from 'react-icons/fa';
+import { VscGitPullRequest, VscGitCommit } from 'react-icons/vsc';
+import { FiUsers } from 'react-icons/fi';
+
+type GitHubStatsProps = {
+  username: string;
+  stats: {
+    prs: number;
+    commits: number;
+    followers: number;
+  };
+  loading?: boolean;
```

Comment:
```suggestion
  loading: boolean;
```
See comment about loading for Leetcode profile